pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -515,8 +515,10 @@ func (m *manager) updateContainerCPUSet(containerID string, cpus cpuset.CPUSet)
 	// this patch-like partial resources.
 	return m.containerRuntime.UpdateContainerResources(
 		containerID,
-		&runtimeapi.LinuxContainerResources{
-			CpusetCpus: cpus.String(),
+		&runtimeapi.ContainerResources{
```

Comment:
note to self: need to carry these param changes forward to the main/containerd/containerd/integration bucket for local testing